CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# Contributing Guidelines
+
+This documentation contains a set of guidelines to help you during the contributing process. We are happy to welcome all the contributors who's willing to enhance/add features to this project. Feel free to contribute and remember, no contribution is too small.
+
+## Submitting Contributions :woman_technologist: :man_technologist:
+
+### Step 1 : Find an issue
+
+- Take a look at the Existing Issues or [create your own Issues!](#creating-an-issue)
```

Comment:
Use `existing`. Avoid unnecessary capitalization.

Reply:
Reword it slightly: `Kindly have a look at the existing issues or create your own issues!`